### ISSUE TITLE
Is it possible to implement yolov8 for image segmentation on hololens2?

### ISSUE THREAD

**Issue by Yuchengli111** [LocalJoost/YoloHolo#12]
Hi!Thank you very much for the article, it helped me a lot, and I would like to ask another question: is it possible to implement the function of yolov8 for image segmentation on hololens2?

**Comment by LocalJoost** [comment]
@Yuchengli111 You mean like this? https://docs.ultralytics.com/tasks/segment/? Possibly, no idea. Be aware HoloLens is a very powerful computer geared toward towards hologram generation, but it does not have a powerful CPU or NPU you can tap into for this kind of things

**Comment by Yuchengli111** [comment]
Yes, thank you very much for your reply. What I want to achieve is indeed the content in the link. I am worried that the performance will be bad...
